apps/server/src/modules/user/legacy/repo/helper/search-query-helper.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { ObjectId } from 'bson';
+import { UserSearchQuery } from '../../interfaces';
+import { SearchQueryHelper } from '.';
+import { RangeType, UsersSearchQueryParams } from '../../controller/dto';
+
+describe('Search query helper', () => {
+	afterEach(() => {
+		jest.resetAllMocks();
+	});
+
+	describe('when search parameters exists', () => {
```

Comment:
There should be a describe block with the name of the tested method around this. Have a look at other tests!